src/sql/parts/insights/browser/insightsDialogView.ts

```diff
@@ -188,8 +189,13 @@ export class InsightsDialogView extends Modal {
 				for (let i = 0; i < this._model.columns.length; i++) {
 					resourceArray.push({ label: this._model.columns[i], value: element.data[i], data: element.data });
 				}
+
 				this._bottomTableData.clear();
 				this._bottomTableData.push(resourceArray);
+				if (bottomTableView.isExpanded()) {
```

Comment:
you might add a comment why we need to collapse and reexpand.  It seems a bit usual to have to do this if `isExpanded()` is returning true.

Reply:
Whats the actual problem here? I'm assuming its a layout issue? in that case there are better ways of recalling layout